Capstone Project

chuaindia opened the pull request:
The following Project requirements are achieved :

- The content of the page is personalized. The topic is a two-day seminar for high school students on **LASER**.

- The pages should look almost identical to the original design. Minor adjustments like text or image changes are acceptable.

- 2 active pages are built :

The home page.
The about page.

- Each of these pages should have versions for 2 different screen sizes:

Mobile: up to 768px wide.
Desktop: 768px or wider.

- In the Mobile menu when the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.

- The mobile menu ave a close (X) button that closes the menu.

- The section "Featured speakers" is created dynamically in JavaScript.

- The Loom video link is attached.

https://www.loom.com/share/9b70ad03440747419c342e8fceaa83a2

chuaindia committed:
Add necessary files and folders for 5th week 2nd day activity

chuaindia committed:
Add new html file for week 5 day2

chuaindia committed:
Add all final files and folders

chuaindia committed:
Add modified js file at 11:20 p.m.

chuaindia committed:
Add modified css file at 11:24 p.m.

chuaindia committed:
Update passwdcheck.js

chuaindia committed:
Delete passwdcheck.js

chuaindia committed:
Update README.md

chuaindia committed:
Merge branch 'main' into capstone_branch

chuaindia committed:
Add modified css file to make linterfree

chuaindia committed:
Update capstone.css

chuaindia committed:
Update capstone.css

chuaindia committed:
Make style file linterfree

chuaindia committed:
Update capstone.css

chuaindia committed:
Merge branch 'main' into capstone_branch

chuaindia committed:
Update capstone.css

chuaindia committed:
Merge branch 'main' into capstone_branch

chuaindia committed:
Update capstone.css

chuaindia committed:
Update capstone.css

chuaindia committed:
Merge branch 'main' into capstone_branch

chuaindia committed:
Update capstone.css

chuaindia committed:
Update capstone.css

chuaindia committed:
Update capstone.css

chuaindia committed:
Merge branch 'main' into capstone_branch

chuaindia committed:
All files and folders

chuaindia committed:
Add style file to get rid of linters

chuaindia committed:
Add new css file to make free from linters

chuaindia committed:
Add new index.html file to make free from linters

chuaindia committed:
Add new about.html file to make free from linters

chuaindia committed:
Add new  file to make free from style linters

chuaindia committed:
Add new CSS file to get rid of stylelinters

chuaindia committed:
Update capstone.css

chuaindia committed:
Update about.html

chuaindia committed:
Update index.html

chuaindia committed:
Delete style.css

chuaindia commented:
I really did not understand why the stylelint Linter is not working. I made the changes several times according to the run error statement. The capstone.css file is ok at main but not working at capstone-branch. How I can get rid of that I didn't understand.
I need help from the reviewer.

chuaindia committed:
Create .eslintrc

chuaindia committed:
Update capstone.css

chuaindia commented:
Again I tried for the last time. I succeeded. But I really did not understand my fault. The whole day I wasted on this work. Though ultimately this will help me to make a "view request" to the reviewer.

BertrandConxy commented on README.md:
```diff
@@ -1,6 +1,6 @@
 # capstone_project
 
-This capstone project will show an online website for a seminar. The content is written by me with provided guidelines. initially two pages named "Home" and "About" is being created.
+This capstone project will show an online website for a seminar. The content is written by me with provided guidelines. initially two pages named "Home" and "About" is being created. At each and every stage of the project it is deployed accordingly with the help of GitHub Page.
 
 # Built With
```
- [x] Please try to use the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and then customise it to your project needs 😃 Make sure that each and every section is covered and by doing this, your README will look professional 😃

chuaindia replied:
Thank you for your valuable comments. 
I will take each of them and will try my best to correct them.

chuaindia replied:
I did it accordingly following Microverse README template.

BertrandConxy commented on README.md:
```diff
@@ -1,6 +1,6 @@
 # capstone_project
 
-This capstone project will show an online website for a seminar. The content is written by me with provided guidelines. initially two pages named "Home" and "About" is being created.
+This capstone project will show an online website for a seminar. The content is written by me with provided guidelines. initially two pages named "Home" and "About" is being created. At each and every stage of the project it is deployed accordingly with the help of GitHub Page.
```
- [ ] The live demo link that you share here in your README does not show up images in the `featured speakers` section. I really don't know exactly the reason why but I do think that it is a matter of the **image paths** or the deployed branch of your project(if it this case, you will need to deploy this branch not the main branch)
 **Live Demo link**
![image](https://user-images.githubusercontent.com/90222110/191791387-b4ce7ef9-6b42-420f-b5c7-7aef2b01f194.png)

**When I run the project locally**
![image](https://user-images.githubusercontent.com/90222110/191792114-9a8c36e1-7691-41e1-8ace-b23b624e682f.png)

so, please try to correct it because your deployed link is what most visitors will be checking 😃

chuaindia replied:
Ok. I will look after that positively.

BertrandConxy commented on capstone.css:
```diff
@@ -445,89 +445,89 @@ footer {
- transition: width 0.5s ease;
- overflow-x: hidden;
- z-index: 5;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 0;
+    background-color: #272a31;
+    color: white;
+    position: fixed;
+    top: 0;
+    left: 0;
+    transition: width 0.5s ease;
+    overflow-x: hidden;
+    z-index: 5;
```
- [ ] By comparing your headline section with the one in the design provided, I can see that there are a lot of things that need to be improved. Please try to view the comparison in the screenshot below and I will try to add specific things to adjust 👇 👇 

**Given design**
![image](https://user-images.githubusercontent.com/90222110/191793513-7df74dce-0f15-4250-a632-d9d2b94eb194.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/191793752-616c8b5a-a409-4833-b2ae-c7b0ff5f2f11.png)


- [x]  Starting with the navbar, there is the logo on the top-left side of the design. Please add the logo to your website too 👍 
- [x] The background color of the headline section is supposed to be `light-grey` which differentiates it with the navbar. PLease implement it on your website.

- [x] Please add the correct color to the active nav link. As it is in the design given, it is expected to have an orange color. Also, the styles on the about link can fool the visitors to think that they are on the About page while they are on the home page. So please correct it by aligning it with the design given
     
![image](https://user-images.githubusercontent.com/90222110/191795180-cf70b8c7-1fee-419a-b8d1-7db0b164f26c.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/191795475-7cb6e7b4-d4c8-4db3-b376-12c0d5345b2e.png)

- [x] The heading text is supposed to be very big compared to others and also has this orange style. Please try to implement it in your design 🏃 
👇 
![image](https://user-images.githubusercontent.com/90222110/191795925-07f5a092-c7ec-4370-9c7e-91ff3634cbaa.png)

I hope that this is very helpful for your design improvements and I look forward to what you can achieve 😃 👍

chuaindia replied:
Yes, I will try to do my best.

chuaindia replied:
I did by increasing the font-size of .firsthead h1

chuaindia replied:
I made the color of the active nav element as required.
I made the headline background-color as "light-grey".

BertrandConxy commented on capstone.css:
```diff
@@ -445,89 +445,89 @@ footer {
 
   .program {
-  height: 930px;
+    height: 930px;
```
- [x] Please try to work on the padding and margin properties of your cards here in the `main program` section so as to match the design given 😎 

**Your design**
![image](https://user-images.githubusercontent.com/90222110/191797674-598f71bc-f2ac-414d-92ac-2e4dcc471300.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/191797848-30ed2034-7eb3-44c2-b307-6de05f71042f.png)

- [x] Also remember to add that encircled link in the given design screenshot 😉

chuaindia replied:
Ok. I will add that. 
Thank you for your vivid review which helps me to enrich my knowledge regarding design.

chuaindia replied:
I did it.

chuaindia replied:
I made the active bar accordingly.

BertrandConxy commented on capstone.css:
```diff
@@ -445,89 +445,89 @@ footer {
- overflow-x: hidden;
+    width: 100%;
+    overflow-x: hidden;
   }
```
- [ ] Please implement the footer in the home page. I have attached some screenshots to show you what I mean 👇 👇 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/191798827-5c700680-7f69-40bc-90af-97634185053d.png)

chuaindia replied:
Ok . I will do that.

chuaindia replied:
I added this in the footer section of about.html.

BertrandConxy commented on capstone.css:
```diff
@@ -445,89 +445,89 @@ footer {
 
   .program {
-  height: 930px;
+    height: 930px;
```
- [x] On the mobile screens, the program cards are supposed to fill the space and align on top of one another. Please try to look at the comparison below 👇 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/191800368-0d9333ba-7968-4e72-9ee5-c7a139022354.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/191800569-60f27ba5-35ce-49b9-ab10-77e1f18d67ed.png)

chuaindia replied:
Ok, I will try definitely.

chuaindia replied:
I am very sorry. I tried it but I am unable to do it.

KayLemba replied:
Please like the previous code reviewer asked please implement this change it should look like the design template you can achieve this by increasing the width of this container

chuaindia replied:
Ok. I am doing that. Thank you for your valuable opinion.

chuaindia replied:
It is done. I am attaching the screenshot.
https://user-images.githubusercontent.com/106825373/192056426-c07d76bd-5d49-4c16-aeb8-2b006cff6e7c.png

BertrandConxy commented on capstone.css:
```diff
@@ -445,89 +445,89 @@ footer {
- grid-auto-flow: row;
- height: 900px;
- overflow-y: hidden;
+    grid-auto-flow: row;
+    height: 900px;
+    overflow-y: hidden;
```
- [x] Please try to improve the featured speakers' section design based on the design given. Take a look at the comparison below 👇 
![image](https://user-images.githubusercontent.com/90222110/191802170-97285071-fa15-49fc-b25c-5e507efb6e6a.png)


![image](https://user-images.githubusercontent.com/90222110/191802251-4036b404-bdc8-45ec-a282-9cbf14f94cd2.png)

chuaindia replied:
Ok. I will follow all of your reviews to correct my capstone project.

Again a lot of thanks to you.

chuaindia replied:
I made the font-size of the names bigger.

BertrandConxy reviewed (changes requested):
Hi @chuaindia ✋ , 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 

- Good use of comments
- No linter errors
- correct github flow is used
- good PR

## Required Changes ♻️

- [x] Great job so far 👏. It looks like you have not set up eslint checks here on GitHub because currently there are four checks while they are supposed to be five. I can see that you have added the `eslint.rc` file but that's not enough. You have to update your `linters.yml` file. Please consider replacing it with this one  [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js/.github/workflows) as provided by Microverse 😉 

- [x] It looks like you have already pushed some codes in the main branch without being reviewed and approved. This is strictly not correct because **Microverse provided code reviewers to make sure that you have met all requirements as required per module** and you are really competent 😎. What I would suggest you, is to check the `main` branch and delete manually everything in the project requirements that you have merged into it (this includes index.html, about.html, and other files except the README) and **then make sure all your changes lie here on this branch. When your Pull request is approved**, then you will be able to merge them into the main branch 😃 👍 

- [x] Your project presentation video is great 🔥 👏 though you forgot to talk about one important point which is the piece of code that you are proud of. This showcases your ability to explain technical concepts 👏 so please record another video and make sure you talk about that point too 😉 
- [x] Please try to improve your about page because I can see that it is missing a lot of section when compared to the design given
**Given design**
![image](https://user-images.githubusercontent.com/90222110/191802753-8705937c-48cc-4f22-ae75-bae7f495876a.png)
![image](https://user-images.githubusercontent.com/90222110/191802831-d93a9b43-5753-4d12-bee4-179ba137ed4e.png)
![image](https://user-images.githubusercontent.com/90222110/191802885-c1a74aa5-e8f6-4e70-a150-3210cf6f8b77.png)


_Check other comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

chuaindia committed:
Update linters.yml

As per the Reviewer's "Required Changes" no. 1, I replaced the linters.yml provided by Microverse following the link given review.

chuaindia committed:
Update capstone.js

Changed the indentation criteria following the instructions shown in the corresponding linter error.

chuaindia committed:
Add new CSS file to overcome linters error

chuaindia committed:
Add files after updation following review

chuaindia committed:
Add new picture files for furnishing the project

chuaindia committed:
Add .js file after removing conflict

chuaindia committed:
Add index.html following the Reviewer's optional requirement

chuaindia commented:
I did not understand the conflicts. Git Bash is showing everything is ok.
I am attaching the screenshots

.
https://user-images.githubusercontent.com/106825373/192022538-68efc5af-2322-40a0-a6f8-a392d3cdab4e.png

https://user-images.githubusercontent.com/106825373/192022627-b911902c-dfdf-497b-b158-894323b549d4.pn

https://user-images.githubusercontent.com/106825373/192022681-63f340a6-c947-4817-adea-102750cc82bc.png

chuaindia committed:
Add capstone.css following the Reviewer's optional requirement

chuaindia committed:
Add js following the Reviewer's optional requirement

chuaindia committed:
Add capstone.css following the Reviewer's optional requirement

chuaindia committed:
Add capstone.css(11:21 p.m.) following the Reviewer's optional requirement

chuaindia commented:
I really don't understand the conflict part. All checks are passed. Gitbash is showing everything up-to-date. So I didn't understand this issue. I have tried my best. The new loom video as required by the Reviewer I am attached here.

Updated LOOM Link : https://www.loom.com/share/80a5e8fb94cf4839b6f9ad304f2317ee

chuaindia committed:
Update README.md

Updated following the Reviewer's review.

KayLemba reviewed (changes requested):
Hi @chuaindia    👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there! Please correctly implement all changes requested by the previous code reviewer in order for to get an approval. 

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

chuaindia commented:
Hi  @KayLemba,

I don't understand what changes again I have to make because I did not see
any comment in the pull review section.
I tried every change needed by the earlier reviewer.
So please guide me.
I am unable to understand your mail.
I am not getting any comments in the pull request section as I got for
every project.

Waiting for your reply.

Thanks,
Tanusri Ghosh.


On Sat, Sep 24, 2022 at 12:18 AM Kalolo Chola Lemba <
***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @chuaindia <https://github.com/chuaindia> 👋
> Good job so far working on the project requirements!
> There are some issues that you still need to work on to go to the next
> project but you are almost there! Please correctly implement all changes
> requested by the previous code reviewer in order for to get an approval.
>
> To Highlight 🎉
> No linter errors. ✔️
> GitHub flow is followed. ✔️
>
> Required Changes ♻️
> Check the comments under the review in the inline comments.
>
> Optional suggestions
> Every comment with the [OPTIONAL] prefix is not crucial enough to stop the
> approval of this PR. However, I strongly recommend you to take them into
> account as they can make your code better.
>
> You can also consider:
> N/A
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.
>
> As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have
> a limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/chuaindia/capstone_project/pull/1#pullrequestreview-1118915900>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZPANHLLSUTZJJALWDGYSA3V7X3OBANCNFSM6AAAAAAQSXDQ4Y>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

KayLemba commented:
> 

Yes you may have changed somethings but they were not properly implemented the previous code reviewers asked for you to make changes which you havent done correctly Your web page design is supposed to match the template design. The previous code reviewer has already pin pointed on what you need to correct in the comments above please make those corrections that were requested by the previous code reviewer because thats were i based my review. You did not implement what you were told to change.

chuaindia commented:
Following the second reviewer, the change is made. I am attaching the screenshot.

https://user-images.githubusercontent.com/106825373/192056919-12a9151a-d65b-47f5-99b8-5f3f7db20be7.png

chuaindia committed:
Update capstone.css

chuaindia commented:
I have made all the reviews cleared as required by the users.

tufoinnkuo10 reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @chuaindia  

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- design well followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

chuaindia commented:
Hi !

A lot of thanks to you .

Thanks,

Tanusri Ghosh

On Sat, Sep 24, 2022 at 3:03 AM TufoinNkuo ***@***.***> wrote:

> ***@***.**** approved this pull request.
> Status: Approved ✔️
>
> Hi @chuaindia <https://github.com/chuaindia>
>
> Your project is complete! Great job for making the requested changes🥇.
> There is nothing else to say other than... it's time to merge it 🍾🚢 [image:
> :shipit:] . Congratulations! 🎉💯🌟
> Good Points 👍
>
>    - Descriptive Pull Request
>    - Validations correctly implemented
>    - design well followed
>    - No linter errors
>
>
> Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have 2
> more limited reviews per this project. If you think that the code review
> was not fair, you can request a second opinion using this form
> <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/chuaindia/capstone_project/pull/1#pullrequestreview-1119060454>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZPANHPAZRGKP54F55HRBJTV7YOZFANCNFSM6AAAAAAQSXDQ4Y>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
